browser/ui/browser_commands.cc

```diff
@@ -259,4 +261,14 @@ void ToggleVerticalTabStripFloatingMode(Browser* browser) {
       !prefs->GetBoolean(brave_tabs::kVerticalTabsFloatingEnabled));
 }
 
+void ToggleActiveTabAudioMute(Browser* browser) {
+  WebContents* contents = browser->tab_strip_model()->GetActiveWebContents();
+  if (!contents || !contents->IsCurrentlyAudible())
```

Comment:
Hmm, we can't assume that only one tab is playing audio. It would be rare but user could do that.
We are thinking to add another shortcut to mute all from active browser window. Maybe that's what you want?
cc @rebron

Reply:
I would think that it would toggle mute on the currently-active tab, whether or not it, or any other tab, is currently playing any sound.

Reply:
I'm tempting to change current `ctrl + m` to toggle all audible tabs(but maybe mostly only one) instead of toggling active tab only.
Although user could have multiple audible tabs but it's rare and that user would not use this `ctrl + m` if it toggles all that tabs.
I think it's more usable when `ctrl + m` toggles any tab as @GregJArnold suggested.
@rebron WDYT?